### ISSUE TITLE
windows build on appveour failing

### ISSUE THREAD

**Issue by rgetz** [analogdevicesinc/libad9361-iio#12]
Currently, the latest appveyor is failing:
https://ci.appveyor.com/project/analogdevicesinc/libad9361-iio

```
appveyor DownloadFile https://ci.appveyor.com/api/projects/analogdevicesinc/libiio/artifacts/libiio-win32.zip?branch=master
Error downloading remote file: One or more errors occurred.
Inner Exception: Remote server returned 404: The specified blob does not exist.
```

That is because that file [doesn't exist anymore](https://github.com/analogdevicesinc/libiio/commit/e1f420ca9eb17535be2ec5b4a59823f7cbd9b0c2), We put the 64, and 32 bit files, and ming in the same zip.

https://ci.appveyor.com/api/projects/analogdevicesinc/libiio/artifacts/libiio.zip?branch=master

This needs an update:
https://github.com/analogdevicesinc/libad9361-iio/blob/master/appveyor.yml

1) to fix the name, and make things build
and
2) to add 32, 64 and ming, just like on libiio

Thanks
-Robin

**Comment by adisuciu** [comment]
Naming fixed with f893d67f478954af11257b38b2bf39719a29e658
Mingw versions added with b03f061d82f9e1a238997d965f51a7d49e2260ee

**Comment by rgetz** [comment]
Thanks - looks like it is working now.
